Jenkinsfile

```diff
@@ -35,6 +91,33 @@ pipeline {
                         | gunzip | sudo tee /usr/local/bin/license-detector > /dev/null
                     sudo chmod +x /usr/local/bin/license-detector 
                 '''
+                 sh '''
+                    if [ ! -d $HOME/google-cloud-sdk/bin ]; then
+                        rm -rf $HOME/google-cloud-sdk
+                        curl https://sdk.cloud.google.com | bash
+                    fi
+                    source $HOME/google-cloud-sdk/path.bash.inc
+                    gcloud components install alpha
+                    gcloud components install kubectl
+                
+                    curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-linux-amd64.tar.gz \
+                        | sudo tar -C /usr/local/bin --strip-components 1 -zvxpf -
+                    curl -s -L https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz \
+                        | sudo tar -C /usr/local/bin --strip-components 1 --wildcards -zxvpf - '*/oc'
+                    
+                    curl -s -L https://github.com/mitchellh/golicense/releases/latest/download/golicense_0.2.0_linux_x86_64.tar.gz \
+                        | sudo tar -C /usr/local/bin --wildcards -zxvpf -
+                  #  curl -s -L https://github.com/src-d/go-license-detector/releases/latest/download/license-detector.linux_amd64.gz \
+                  #      | gunzip | sudo tee /usr/local/bin/license-detector > /dev/null
+                    curl -s -L https://github.com/src-d/go-license-detector/releases/download/v3.0.2/license-detector.linux_amd64.gz \
+                        | gunzip | sudo tee /usr/local/bin/license-detector > /dev/null
+                    sudo chmod +x /usr/local/bin/license-detector 
```

Comment:
Why do we need to install it twice?

Also do you need openshift-origin-client-tools ?

Reply:
https://github.com/Percona-Lab/percona-dbaas-cli/pull/105/commits/1e1c15c33a8ef79ed2b740094940a28faefc1e1a

yes we need openshift-origin-client-tool